### PR TITLE
[Docker] tensorflow_aarch64 package upgrade

### PR DESCRIPTION
**Pull request by neildhickey**
The old version was no longer available so upgrading to 2.9.3

### CODE REVIEW

**Comment by Liam-Sturge** [comment]
LGTM @neildhickey , and should fix the error that started happening on [The Daily Docker Builds](https://ci.tlcpack.ai/blue/organizations/jenkins/docker-images-ci%2Fdaily-docker-image-rebuild/detail/daily-docker-image-rebuild/780/pipeline/64)

**Review by Mousius — APPROVED** [review]

**Review by ekalda — APPROVED** [review]
LEGITIMATE

File: docker/install/ubuntu_install_tensorflow_aarch64.sh [hunk 1]
```diff
@@ -26,4 +26,4 @@ apt-install-and-clear -y --no-install-recommends libhdf5-dev
 pip3 install \
     numpy==1.23.5 \
     keras==2.9 \
-    tensorflow-aarch64==2.9.1
+    tensorflow-aarch64==2.9.3
```
Comment (lhutton1) [inline docker/install/ubuntu_install_tensorflow_aarch64.sh:29]: We could use `~=` to make sure patch updates are automatically used in the future

**Review by lhutton1 — APPROVED** [review]
thanks!

**Pull request merged**